test/ruleset/rules/v2/asyncapi2-message-examples.spec.ts

```diff
@@ -345,12 +345,24 @@ testRule('asyncapi2-message-examples', [
                   {name: 'speed', type: 'string'}
                 ]
               },
+              headers: {
+                type: 'record',
+                name: 'TestHeader',
+                fields: [
+                  {name: 'someKey', type: 'string'},
+                  {name: 'someOtherKey', type: 'string'}
+                ]
+              },
```

Comment:
What was this change for? 🤔

Reply:
I think I added this when the same tests were being used for both rules, to ensure no change in behaviour, less relevant now, can remove